app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -82,7 +82,9 @@ export default {
     },
     FILTER_HEADERS() {
       return this.versionSelected !== ''
-        ? this.formFields.map((f) => ({ name: f, value: f }))
+        ? this.formFields.length > 0
```

Comment:
This allows the formFields to be empty as sometimes when generating the headers for a submission, it can be empty because there are no submissions. In theory, it should contain the form fields from the form design but this is a separate issue.